scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -1961,7 +2005,7 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
       case _: Xml.Start =>
         xmlPat()
       case _: `(` =>
-        makeTuplePatParens(noSeq.patterns())
+        makeTuplePatParens()
```

Comment:
This change is, I guess, the reason for the change in the signature of `makeTuplePatParens`. So I'm just trying to understand the motivation. The thing is that in the current shape `makeTuplePatParens` is different from other make-tuple functions, so it'd be nice to have a good reason for such a discrepancy.